app/views/admin/banners/_form.html.erb

```diff
@@ -54,7 +54,22 @@
     <div class="small-12 column">
       <%= f.label :sections, t("admin.banners.banner.sections_label") %>
       <%= f.collection_check_boxes(:web_section_ids, @banner_sections, :id, :name) do |b| %>
-        <%= b.check_box %> <%= t("admin.banners.banner.sections.#{b.text}") %><br>
+        <%= b.check_box %> <%= t("admin.banners.banner.sections.#{b.text}") %><br> 
+		<% if b.text == 'budgets'%>
+		  <% Budget::Group.all.each do |group| %>
+		    <% if group.budget.phase != 'finished' %>
+		      <span style="display:inline-block; width: 20px;"></span><%= group.name %><br>
+			  <% group.headings.each do |heading| %>
+		        <span style="display:inline-block; width: 40px;"></span>
+	  		    <% if (@banner.headings!=nil) && (@banner.headings.include? heading.id.to_s) %>	  
+	              <%= check_box_tag "banner_headings[]",heading.id, checked = true, id: "banner_headings_#{heading.id}" %> <%= heading.name %><br>
+	  		    <% else %>
+	              <%= check_box_tag "banner_headings[]",heading.id, checked = false, id: "banner_headings_#{heading.id}" %> <%= heading.name %><br>		  
+	  		    <% end %>
+			  <% end %>	
+			<% end %>  
+		  <% end %>	
+		<% end %> 
```

Comment:
I think we always use two spaces for indentation; never tabs, and never four spaces. It would be great if you could review this pull request and adjust the indentations as well as trailing whitespaces and newline characters at the end of a file :+1:.